pack/mts_encounter.json

```diff
@@ -523,6 +532,7 @@
 		"set_code": "tower_defense",
 		"set_position": 1,
 		"stage": 1,
+		"text": "<b>Proxima Midnight cannot be defeated while Corvus Glaive has any hit points remaining.</b>",
```

Comment:
I think you're missing the `attack_text` on this card:

```json
"attack_text": "<b>Forced Interrupt:</b> When Proxima Midnight attacks you, choose to either deal 1 damage to Avenger's Tower, or Proxima Midnight gets +2 ATK for this attack."
```

Reply:
added in, thanks!